### DISCUSSION TITLE
Well-known URLs and redirection

### DISCUSSION THREAD

**Discussion by matatk** [w3c/personalization-semantics#253] [Ideas]
@abhina-kumar asks:

> Do we want to have application side (redirection on calling well-known url) of well-known URL also part of the specification?

My understanding is that the Well-known URL is really for the machine, so yes - visiting the well-known URL would redirect to the "real" page on the site. Does that answer your question?

We should check how this works in the wild with `change-password` and instances of other registered URLs.

**Comment by abhina-kumar** [comment]
> @abhina-kumar asks:
> 
> > Do we want to have application side (redirection on calling well-known url) of well-known URL also part of the specification?
> 
> My understanding is that the Well-known URL is really for the machine, so yes - visiting the well-known URL would redirect to the "real" page on the site. Does that answer your question?
> 
> We should check how this works in the wild with `change-password` and instances of other registered URLs.

@matatk :
Can we have some sequence diagram for the interaction. As the proposal also contains well-known url for sub-pages, how the machine (plug-in, browser etc.) gets to know that there is well-known url at this page?

**Comment by GiaGlad** [comment]
I have a website page example.com/abc and want to redirect example.com/abc. and thsi url is 15 years old. 

Is it a good idea to change the url? Will it help me in ranking? Let me know your thoughts.
